README.md

```diff
@@ -63,9 +69,6 @@ Keep a note of your cluster name. Alternatively, you can also go to your [hasura
 
 ### Step 3: Deploying on a hasura cluster
 
-1. Open the package.json file at `microservices/ui/app/`
```

Comment:
These are required, and are present here https://github.com/hasura/hello-react/blob/master/microservices/ui/app/package.json#L17

Reply:
Oh that's weird. When I replaced the `app` with my existing app, I overwrote the `package.json` with my app's. Despite that, the deployment works perfectly. Doesn't it imply that those lines are redundant?

Reply:
This is because you have hard coded your cluster name inside your react app, yes ? An ideal case would be to have the cluster name added as an env variable to your app, so that you can easily move between your dev and prod cluster (by having different npm scripts for each)

Reply:
I thought the quickstart command creates the cluster and uses the name as an env var. I don't recall hardcoding the cluster name tbh.
Moreover, aren't those instructions at focus actually telling us to hardcode them?

Reply:
Is your react app using create-react-app ?

Reply:
yes, pretty much.